lsp-mode.el

```diff
@@ -5224,10 +5293,19 @@ It will show up only if current point has signature help."
       (funcall action-handler action)
     (lsp--send-execute-command command arguments?)))
 
-(lsp-defun lsp-execute-code-action ((action &as &CodeAction :command? :edit?))
+(lsp-defun lsp-execute-code-action ((action &as &CodeAction :command? :edit? :disabled?))
```

Comment:
Should this have an optional `kinds` argument? Generally, it seems like almost all the functions that operate on code actions could take an optional `kinds` argument, perhaps that would let us get away with fewer functions.

Reply:
Not in this case. `lsp-execute-code-action` should execute whatever action it is given. It is only additionally interactive.